spec/features/admin/forms_spec.rb

```diff
@@ -728,37 +728,6 @@
               click_on "Update Question option"
             end
 
-            it "click through to Edit page" do
-              expect(page).to have_content("Question option was successfully updated.")
-              within (".question") do
-                expect(page).to have_content("Edited Question Option Text")
-              end
-
-              # Ensure other (non UI-visible) data persists
-              visit edit_admin_form_question_question_option_path(form, radio_button_question, radio_button_option)
-              expect(page.find_field("question_option_text").value).to eq("Edited Question Option Text")
-              expect(page.find_field("question_option_value").value).to eq("100")
-            end
-          end
-
-          describe "edit Dropdown option" do
```

Comment:
this spec is still valid.
the user will still be able to edit the question option, but in a different way UI-wise.